features/facebookImport/src/model/analyses/ministories/messages-analysis.js

```diff
@@ -7,11 +7,11 @@ export default class MessagesAnalysis extends RootAnalysis {
     }
 
     async analyze({ dataAccount }) {
-        let messagesCount = dataAccount.messagesCount;
+        let messagesCount = dataAccount.messages?.messagesCount || 0;
```

Comment:
Nit: Seems like `??` would be a little less error prone than `||` here - though it works as long as the alternative value is `0`.